packages/codemods/src/v8-v9/transforms/dropdowns.spec.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable jest/require-hook */
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+//  https://github.com/facebook/jscodeshift/blob/main/README.md#unit-testing
+// @ts-expect-error - TS7016: Could not find a declaration file for module
+import { defineSnapshotTest } from 'jscodeshift/dist/testUtils';
```

Comment:
Not types are available. 😞 

Would it be better to add `declare module jscodeshift/dist/testUtils {}` to a `.d.ts` file?